nms/app/packages/magmalte/app/views/events/EventsTable.js

```diff
@@ -214,11 +219,75 @@ type EventTableProps = {
   inEndDate?: moment,
 };
 
+const defaultProps = {
+  shouldAutorefresh: true,
+};
+
+type UseRefreshingDateRangeHook = (
```

Comment:
Move this hook and the type to a separate file under components directory, so that it can be shared across the other components in the future.